src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -108,51 +109,46 @@ const DOCS: {[key: string]: DocCategory[]} = {
   ],
   [CDK] : [
     {
-      id: 'components',
-      name: 'Components',
-      items: [
-        {id: 'table', name: 'Table', examples: []},
-        {id: 'stepper', name: 'Stepper', examples: []},
-
-      ]
-    },
-    {
       id: 'component-composition',
-      name: 'Component Composition',
+      name: 'Common Behaviors',
       items: [
+        {id: 'a11y', name: 'Accessibility', examples: []},
         {id: 'observers', name: 'Observers', examples: []},
         {id: 'layout', name: 'Layout', examples: []},
         {id: 'overlay', name: 'Overlay', examples: []},
         {id: 'portal', name: 'Portal', examples: []},
         {id: 'bidi', name: 'Bidirectionality', examples: []},
         {id: 'scrolling', name: 'Scrolling', examples: []},
-        {id: 'viewport', name: 'Viewport', examples: []},
-      ]
-    },
-    {
-      id: 'utilities',
-      name: 'Utilities',
-      items: [
-        {id: 'coercion', name: 'Coercion', examples: []},
-        {id: 'collections', name: 'Collections', examples: []},
-        {id: 'keycodes', name: 'Keycodes', examples: []},
-        {id: 'platform', name: 'Platform', examples: []},
-        {id: 'rxjs', name: 'RxJS', examples: []},
       ]
     },
     {
-      id: 'accessibility',
-      name: 'Accessibility',
+      id: 'components',
+      name: 'Components',
       items: [
-        {id: 'focus-key-manager', name: 'Focus Key Manager', examples: []},
-        {id: 'focus-trap', name: 'Focus Trap', examples: []},
-        {id: 'interactivity-checker', name: 'Interactivity Checker', examples: []},
-        {id: 'list-key-manager', name: 'List Key Manager', examples: []},
-        {id: 'live-announcer', name: 'Live Announcer', examples: []},
+        {id: 'table', name: 'Table', examples: []},
+        {id: 'stepper', name: 'Stepper', examples: []},
+
       ]
     },
+    // TODO(jelbourn): re-add utilities and a11y as top-level categories once we can generate
+    // their API docs with dgeni. Currently our setup doesn't generate API docs for constants
+    // and standalone functions (much of the utilities) and we have no way of generating API
+    // docs more granularly than directory-level (within a11y) (same for viewport).
   ]
 };
+
+for (let category of DOCS[COMPONENTS]) {
```

Comment:
You could probably combine these loops into one w/ something like this:

```javascript
for (const type in DOCS) {
  for (const cat of DOCS[type]) {
    for (let doc of cat.items) {
      doc.packageName = cat;
    }
  }
}
```

Reply:
The `packageName` isn't the category; the categories are things like `forms`, `layout`, etc.

The `type` _also_ isn't the package name, it's the _section_ name ("components" and "cdk").

Reply:
What I really want to do is generate this entire blob.